heron/executor/src/python/heron_executor.py

```diff
@@ -506,7 +506,8 @@ def _get_healthmgr_cmd(self):
                      "--role", self.role,
                      "--environment", self.environment,
                      "--topology_name", self.topology_name,
-                     "--metricsmgr_port", self.metrics_manager_port]
+                     "--metricsmgr_port", self.metrics_manager_port,
+                     "--verbose"]
```

Comment:
suggest moving this line to a separated PR.

Reply:
updated